src/js/view/controlbar.js

```diff
@@ -769,6 +782,11 @@ define([
             _model.setVolume(pct * 100);
         }
 
+        function _seekThrottled(pct) {
+            _seekTo = pct;
+            _seekThrottler(pct);
```

Comment:
So we're still doing a combination of throttled requests with requests made on slider time events. Is that to support js and flash? ideally the first request is immediate - 'dragging' begins, and while dragging seek requests are made on seeked events.